src/shell/dotnet/Shell/Shell.csproj

```diff
@@ -14,7 +14,13 @@
 		<None Remove="MainWindow.xaml" />
 		<None Remove="WebWindow.xaml" />
 	</ItemGroup>
-
+	
+	<ItemGroup>
+	  <EmbeddedResource Include="Preload\fdc3-iife-bundle.js">
+	    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	  </EmbeddedResource>
+	</ItemGroup>
+	
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Extensions.Hosting" Version="7.0.1" />
 		<PackageReference Include="Microsoft.Web.WebView2" Version="1.0.1462.37" />
```

Comment:
Not related to this PR, but do we have dependapot configured for nuget packages? We could use a more recent version of the WebView2 SDK.

Reply:
Yes but I think we can't configure with root so it isn't seeing any of the sln/projects.  We might have to have an entry for each sln

https://github.com/morganstanley/ComposeUI/blob/main/.github/dependabot.yml#L12

Reply:
We had a few dependabot PRs for Nuget last year:
https://github.com/morganstanley/ComposeUI/pull/110
But this project might have been in the root at that time and that's why it was picked up.
Let me revisit the config in a separate PR.